Tasks/Viachaslau/Task3/query06.sql

```diff
@@ -0,0 +1,9 @@
+SELECT CustomerID,
+       T1.OrderDate
+FROM (
+       SELECT CustomerID,
+              MAX(YEAR(OrderDate)) OVER (PARTITION BY CustomerID) AS MAXYEAR,
```

Comment:
it calculate that latest year was 2011 and filter by it
however if there would be records in 2010 they would be in the result (try same with 2012 and you'll see difference)
also PARTITION BY waste a lot of resources but useless here